Controller/GroupController.php

```diff
@@ -32,7 +32,7 @@
  * @author Thibault Duplessis <thibault.duplessis@gmail.com>
  * @author Christophe Coevoet <stof@notk.org>
  */
-class GroupController extends Controller
+final class GroupController extends AbstractController
```

Comment:
We should first make a 2.x release tagging them all as `@final` so that the DebugClassLoader can give a deprecation warning to projects extending controllers.

Reply:
Good idea: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2951